tests/validation_suite/automation_framework/worker/worker_params.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Worker.py -- functions to perform worker related functions based on Spec 1.0 compatibility
```

Comment:
This file look exactly same as that in Avalon source code. If there is no change directly use WorkerDetails/SGXWorkerDetails from Avalon source instead of duplicating.

Reply:
> This file look exactly same as that in Avalon source code. If there is no change directly use WorkerDetails/SGXWorkerDetails from Avalon source instead of duplicating.

Removed the duplicate file and referenced class from source